hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java

```diff
@@ -537,4 +547,24 @@ public int getMaxReturnedNftsPerTouch() {
 	public Gas exchangeRateGasReq() {
 		return exchangeRateGasReq;
 	}
+
+	public double getStakingRewardRate() {
```

Comment:
Why is this a `double`?  From the HIP,

> `StakingRewardRate` - the total tinybars earned as staking reward during each staking period (example: 100_000_000L * 1_000_000_000 / 365 to emit rewards at a total rate of one billion hbars per year)

...seems like a `long` would be more appropriate?

Reply:
ahh, I modified this in the working branch but not here.. Will update this.